README.md

```diff
@@ -153,8 +153,19 @@ Returns an object with the following properties:
 By default, it will search for moon rise and set during local user's day (frou 0 to 24 hours).
 If `inUTC` is set to true, it will instead search the specified date from 0 to 24 UTC hours.
 
+### Solar time
+
+```js
+SunCalc.getSolarTime(/*Date*/ date, /*Number*/ latitude, /*Number*/ utcOffset)
```

Comment:
Based on the original source code, it appears this should use longitude.
Second parameter should be utcOffset and third parameter should be Longitude.

Reply:
Fixed. I've been trying different ways to achieve the same. Missed the readme. Thanks.